projects/media-viewer/src/lib/toolbar/sub-toolbar/sub-toolbar.component.html

```diff
@@ -1,64 +1,49 @@
-<div id="secondaryToolbar" class="secondaryToolbar doorHangerRight"
-     [hidden]="toolbarEvents.subToolbarHidden | async">
+<div id="secondaryToolbar" class="secondaryToolbar doorHangerRight" [hidden]="toolbarEvents.subToolbarHidden | async">
```

Comment:
is the subtoolbar still needed given you're using the overlay? maybe you could reuse it for the overlay stuff

Reply:
The subtoolbar will be removed as part of the next commit HOWEVER the tests relating to the zoom, print, download and rotate will be moved to the maintoolbar as these are still required. Note the secondary button tests are no longer required as there is only one toolbar driven by the template.